src/components/ProfileModal/index.tsx

```diff
@@ -51,9 +57,13 @@ function ProfileModal({ openShowModal, setOpenShowModal }: ModalProps) {
     )
     .then(res => {
       console.log(res);
-      // res.data?.errors
-      // ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
-      // : setReqStatus({success: "Movie created successfully :D"})
+      if(reqStatus !== null) {
+        return;
+      } else {
+        res.data.errors !== undefined 
+          ? setReqStatus({error: {errorMessage: res.data.errors[0].message}}) 
+          : setReqStatus({success: 'User updated successfully'});
```

Comment:
Should it be something like "{success:`${itemToUpdate} updated successfully`});"? @Hai5edfm